.eslintrc.json

```diff
@@ -0,0 +1,271 @@
+{
+    "env": {
+        "es6": true
```

Comment:
How so? It defines ES2015 globals like `Set` or `Promise`. Wed don't want that here as some of our supported environments don't have those globals.